patches/api/0387-Add-EntityStopTickingEvent.patch

```diff
@@ -0,0 +1,58 @@
+From 0000000000000000000000000000000000000000 Sun Jul 17 00:00:00 2001
+From: Oliver Kent <support@newwindserver.com>
+Date: Sun, Jul 17 2022 13:57:33 -0100
+Subject: [PATCH] Add EntityStopTickingEvent
+
+Adds a new event that is called when an entity stops ticking.
+
+diff --git a/src/main/java/net/minecraft/server/level/ServerLevel.java b/src/main/java/net/minecraft/server/level/ServerLevel.java
+index 1a918eabb..4429d36ff 100644
+--- a/src/main/java/net/minecraft/server/level/ServerLevel.java
++++ b/src/main/java/net/minecraft/server/level/ServerLevel.java
+@@ -2512,6 +2512,10 @@ public class ServerLevel extends Level implements WorldGenLevel {
+                 pearl.ownerUUID = null;
+             }
+             // Paper end
++
++            // Paper start - Call EntityStopTickingEvent
++            getCraftServer().getPluginManager().callEvent(new io.papermc.paper.event.entity.EntityStopTickingEvent(entity.getBukkitEntity()));
++            // Paper end
+         }
+ 
+         public void onTrackingStart(Entity entity) {
+
```

Comment:
👋 Welcome to Paper, congrats on making your first PR!

This section should be in a server patch, since the implementation doesn't exist as far as the API is concerned.